src/app/services/raiden.service.ts

```diff
@@ -296,15 +312,21 @@ export class RaidenService {
                     body
                 );
             }),
+            map((channel: Channel) => {
+                channel.settle_timeout = (<BigNumber>(
```

Comment:
Ok I am a little bit confused here why do we need to double cast and convert to a number? reveal timeout/settle timeout should be a normal number is that done due to the interceptor?

Reply:
Yes, the interceptor converts all numbers to BigNumbers. So the To prevent this for specific object keys we would need to pass parameters to the interceptor. But that is not possible by default.